cmd/envtool/envtool.go

```diff
@@ -188,6 +217,7 @@ func setup(ctx context.Context, logger *zap.SugaredLogger) error {
 		setupPostgresSecured,
 		setupMongodb,
 		setupMongodbSecured,
+		setupMySQL,
```

Comment:
```suggestion
		setupMySQL,
		setupMongodb,
		setupMongodbSecured,
```

To keep the same general order